cmd/root.go

```diff
@@ -50,8 +50,20 @@ const (
 	msgSize = 512 * 1024 * 1024
 )
 
-var encodings = []string{"json", "bytes", "proto", "ascii", "json_ietf"}
-var formats = []string{"json", "protojson", "prototext", "event", "proto"}
+var encodings = [][2]string{
+	{"json", "indicates that the value is encoded as a JSON string,as specified in RFC7159"},
+	{"bytes", "contains a byte sequence whose semantics is opaque to the protocol"},
+	{"proto", "contains a serialised protobuf message using protobuf.Any"},
+	{"ascii", "an ASCII encoded string representing text formatted according to a target-defined convention"},
+	{"json_ietf", "similar to `json` with aditional types defined"},
```

Comment:
JSON_IETF encoded string (RFC7951)

Reply:
is it not this RFC ? https://tools.ietf.org/html/rfc7493